stats/lib/stats-fetchers.js

```diff
@@ -270,3 +270,48 @@ export const fetchDailyRetrievalResultCodes = async (pgPools, filter) => {
   const stats = Object.entries(days).map(([day, rates]) => ({ day, rates }))
   return stats
 }
+
+/**
+ * Fetches daily global retrieval time statistics
+ * @param {import('@filecoin-station/spark-stats-db').PgPools} pgPools
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchDailyRetrievalTimes = async (pgPools, filter) => {
+  const { rows } = await pgPools.evaluate.query(`
+    SELECT
+      day::text,
+      percentile_cont(0.5) WITHIN GROUP (ORDER BY time_to_first_byte_p50) AS ttfb_p50
```

Comment:
`percentile_cont` returns a double precision floating number, e.g. `20.124`. This is likely to create a sense of false precision.

Let's convert the number to an integer, e.g. using the Postgres `CEIL()` function.

- We could also use `ROUND()` (10.2→10, 10.8→11), but I prefer to be conservative in what we produce and prefer to report slightly worse TTFB values rather than slightly better values.

This is a minor detail, though - I am fine with any rounding function. 

What's important to me is to produce integer values (number of milliseconds), since that's the precision we are getting from the checker ndoes.

Reply:
I have missed this as I was testing using round int values. I have added rounding to queries and we're now also rounding computed p50 values before inserting them to the database (in both cases we're rounding using ceil).

I am going to change tests in https://github.com/filecoin-station/spark-evaluate/pull/434 to include more odd values.